scripts/BusFruCallouts.pm

```diff
@@ -0,0 +1,294 @@
+# IBM_PROLOG_BEGIN_TAG
```

Comment:
This is required since processMrw has a dependency on this Perl module.
FYI, This module was newly added in the HB, and during migration, we need to pull that as well.